### DISCUSSION TITLE
Unable to view ICONS on IOS Device

### DISCUSSION THREAD

**Discussion by rksambari** [google/model-viewer#3937] [Q&A]
Hi,

I have added a few ICONS in my code and they are working perfectly fine when I open with an Android device, but those ICONS are not visible when opened with IOS Device.
FYI, I have used AR Modes **webxr** and **quick-look**.
Please refer to the attached image taken from android(ICONS right side, youtube icons).
![AR](https://user-images.githubusercontent.com/99704624/201016206-fd5f0995-f5cf-4a23-949a-c6bc53f972d4.jpeg)

**Comment by danilokrulj** [comment]
From what I know you can't add pretty much anything accept some banner in quick-look. At least in AR.

**Comment by elalish** [comment]
Indeed, please petition Apple to support WebXR so iOS can have as good an AR experience as Android (@grorg).

**Reply by rksambari** [reply]
Most of our customers are IOS users, so is there any work around for IOS?

**Reply by danilokrulj** [reply]
Until they support WebXR on iOS i think all you can do is move the icons/buttons to be seen/used before entering AR, since you have almost no control in quick-look. This way at least you can display all of the content for both users (android and iOS) before entering AR.
